src/styles/_custom_variables.scss

```diff
@@ -61,7 +61,20 @@
   --ds-notification-bg-danger: #{darken(adjust-hue($danger, -10), 10%)};
   --ds-notification-bg-info: #{darken(adjust-hue($info, -10), 10%)};
   --ds-notification-bg-warning: #{darken(adjust-hue($warning, -10), 10%)};
-
+	--ds-notification-success-color: #307B23;
+  --ds-notification-danger-color: #9a6e6e;
+  --ds-text-warning-color: #cf822c;
+  --ds-text-success-color: #74a030;
+  --ds-range-filter-border-color: #949494;
+  --ds-range-filter-connect-color: #63852e;
+	
```

Comment:
Trailing whitespace here. This should simply be a newline, not a newline and a space/tab.